### ISSUE TITLE
Game Crashes on Start Server in Pop!_os

### ISSUE THREAD

**Issue by FoolishDave** [OpenRCT2/OpenRCT2#12924]
**OS:** Pop!_OS 
**Version:** 20.04 LTS
**Commit/Build:** 0.3.0-develop-0\~1810-202009120011-ged9c78557\~ubuntu20.04.1

Game becomes unresponsive when trying to start a server on Pop!_os 20.04 LTS, I have not tried other versions of the OS and joining multiplayer servers and playing singleplayer works fine.

- [ ] Reproducible in RCT2 (vanilla)?
- [x] Specific to multiplayer?

**Steps to reproduce:**
1. Install OpenRCT2 as per the Debian/Ubuntu/Mint quickstart.
2. Attempt to start multiplayer server.

**Dump file**

**Screenshots / Video:**

**Save game:**

**Comment by James103** [comment]
Have you tried running `openrct2 --verbose`, piping the log output to a file, and the uploading that file as a log file?

**Comment by janisozaur** [comment]
a symbolicated stack trace would be more useful.

**Comment by janisozaur** [comment]
Also, which one is it, unresponsive or crashing?

**Comment by FoolishDave** [comment]
Ah my mistake, I'll attach the log files. After pressing Start Server I hear a window open, hear a notification sound, and the game becomes unresponsive and I have to force quit, the logs seem to indicate that nothing is happening after this point as I stop getting anything on stdout or stderr after the IPV6_V6ONLY failed. 92 error.

I attempted to use the fix described in issue [#5212](https://github.com/OpenRCT2/OpenRCT2/issues/5212) but it seemed to yield the same problem (I've attached the logs for both.)

[rctlogs.log](https://github.com/OpenRCT2/OpenRCT2/files/5214969/rctlogs.log)
[rctlogswithspecifiedaddress.log](https://github.com/OpenRCT2/OpenRCT2/files/5214970/rctlogswithspecifiedaddress.log)
